### DISCUSSION TITLE
Color correction node

### DISCUSSION THREAD

**Discussion by saleepeppe** [olive-editor/olive#1836] [Q&A]
Is the color correction node missing in the current nighly build? Is there a build which has it fully functional?

Thanks in andvance and for the amazing job you are doing with the editor!!!

**Comment by DaniSeeh** [comment]
I think many of us deeply desire excellent color grading tools in Olive.

Honestly for the type of occasional freelance stuff I do right now (weddings) it's all that is missing from Olive for me.

I'm sure Matt will implement color tools soon.

He said in his patreon update that a lot of new features should be coming in the next few months because he feels like the architecture is getting there. I for one am very much looking forward to those features. It also looks like he might be adding back an
 effects menu that will deal with the node backend for you in the parameter editor. If that is what those buttons will be used for then I'm also in favor of that for the users who refuse to learn nodes or just want to quickly throw something together and don't want to have to dig into the node editor even if they do understand how to do so.
